yadro/watcher.cpp

```diff
@@ -194,6 +272,28 @@ void dbuswatcher::powerStateChanged(int prev)
     TRACE("Host power state change value: %d -> %d\n",
           prev, hostPowerState);
 }
+/**
+ * @brief Called when BMC Version changed.
+ *
+ * @param prev - previous version value
+ */
+void dbuswatcher::versionBMCChanged(const std::string& prev)
+{
+    TRACE("BMC Version changed: '%s' -> '%s'\n",
+            prev.c_str(),
+            versionBMC);
+}
+/**
+ * @brief Called when BIOS Version changed.
+ *
+ * @param prev - previous version value
+ */
+void dbuswatcher::versionBIOSChanged(const std::string& prev)
+{
+    TRACE("BIOS Version changed: '%s' -> '%s'\n",
+            prev.c_str(),
```

Comment:
why use `std::string` as argument if you anyway only use it as C-string? Why not use `const char *` ?